docs/_tutorials/moe-inference-tutorial.md

```diff
@@ -35,19 +47,17 @@ ds_engine = deepspeed.init_inference(model,
                                      mp_size=tensor_slicing_size,
                                      dtype=torch.half,
                                      moe_experts=args.num_experts,
-                                     checkpoint==getattr(args, 'checkpoint_path')
+                                     checkpoint=getattr(args, 'checkpoint_path')
```

Comment:
@RezaYazdaniAminabadi, is this `getattr` missing a default? currently it would just raise a AttributeError if checkpoint_path isn't in the args.